### PR TITLE
Pt/add torch admin

### PR DESCRIPTION
**Pull request by priscilatrevino**
Installed torch to perform CRUD operations on ecto models. Added tables into our db that matches our data model. 
Torch Admin Interface

### CODE REVIEW

File: lib/lunchbot/lunchbot_data.ex [hunk 1]
```diff
@@ -0,0 +1,1579 @@
...
+  def change_user(%User{} = user, attrs \\ %{}) do
+    User.changeset(user, attrs)
+  end
+
+  defp filter_config(:users) do
+    defconfig do
+      text(:email)
+      text(:role)
+    end
+  end
```
Comment (RVRX) [inline lib/lunchbot/lunchbot_data.ex]: Compiler linter is asking for all definitions of `filter_config/1` to be grouped together:
```
warning: clauses with the same name and arity (number of arguments) should be grouped together, "defp filter_config/1" was previously defined
```
I just put them all at the end of the file when I implemented torch on helios

File: lib/lunchbot_web/templates/layout/torch.html.heex [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    <!--<link phx-track-static rel="stylesheet" href={Routes.static_path(@conn, "/torch/base.css")}>-->
+    <link phx-track-static rel="stylesheet" href={Routes.static_path(@conn, "/torch/theme.css")}>
+  </head>
+
+  <body class={Torch.PageView.body_classes(@conn)}>
+    <header>
+      <section id="torch-account-info">
+        <div class="torch-container">
+          <a href="/">user@example.com</a>
+          <a href="/">Logout</a>
```
Comment (RVRX) [inline lib/lunchbot_web/templates/layout/torch.html.heex:19]: Some defaults here but I guess that's fine for now, lets just remember to either incorporate these into the auth system once that's merged, or delete them.

**Review by RVRX — APPROVED** [review]
All tests pass and it works fine 👍🏻

File: lib/lunchbot/lunchbot_data.ex (hunk 1 above)
Comment (priscilatrevino) [reply lib/lunchbot/lunchbot_data.ex]: Ok thanks, I'll give that a try and hopefully that gets rid of the warning.

**Pull request merged**